### DISCUSSION TITLE
How train hifi-gan after training Glowtts models

### DISCUSSION THREAD

**Discussion by sulaiman-kagumire** [coqui-ai/TTS#2319] [General Q&A]
Hello, Im just new to NLP and trying to develop a tts model for a new language. I have used a  pretrained glowtts to train.
Do I need to train a vocoder to produce waveforms? if yes, how do  I proceed with training a hifi-gan vocoder?
or is glowtts alone enough to get the waveforms without need for a vocoder?

**Comment by augensternHe** [comment]
I have the same problem, thanks for bringing it up

**Comment by erogol** [comment]
You can check `train_hifigan.py` recipe in recipes
